yoti-sdk-impl/src/main/java/com/yoti/api/client/docs/session/create/filters/document/SimpleDocumentRestriction.java

```diff
@@ -0,0 +1,30 @@
+package com.yoti.api.client.docs.session.create.filters.document;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+class SimpleDocumentRestriction implements DocumentRestriction {
+
+    @JsonProperty("country_codes")
+    private List<String> countryCodes;
+
+    @JsonProperty("document_types")
+    private List<String> documentTypes;
```

Comment:
I believe these should be `null` (or excluded) when the list is empty

Reply:
I've just referred to the Docs and it looks like empty arrays are excluded with `@JsonInclude(JsonInclude.Include.NON_EMPTY)` annotation